.github/workflows/publish_npm.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [published]
```

Comment:
Would trigger on prereleases, might not be an issue for you though. If you wanted to only run this on releases, do
``` suggestion
    types: [released]
```